worker_heartbeat.py

```diff
@@ -0,0 +1,28 @@
+import json, urllib, urllib2, time
```

Comment:
Please add brief comment with example usage, even though it seems painfully obvious now; some other build scripts we've had (e.g. updating the tutorials) ended up having non-obvious usage patterns so now I'm wary